sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsActions.java

```diff
@@ -56,7 +57,8 @@ public TextAnalyticsActions setDisplayName(String displayName) {
      * @return the {@link TextAnalyticsActions} object itself.
      */
     public TextAnalyticsActions setRecognizeEntitiesOptions(RecognizeEntitiesOptions... recognizeEntitiesOptions) {
-        this.recognizeEntitiesOptions = Arrays.asList(recognizeEntitiesOptions);
+        this.recognizeEntitiesOptions = recognizeEntitiesOptions == null ? null
```

Comment:
the behavior that was here before was making `this.recognizeEntitiesOptions` null too? or an empty list?

Reply:
this changes is for not to cause NPE in Arrays.asList(notNull). The default value for `this.recognizeEntitiesOptions` is null. We don't want to sent an empty list to service. which means different than sent the null.
```
@SafeVarargs
    public static <T> List<T> asList(T... a) {
        return new Arrays.ArrayList(a);
    }
```

Reply:
to make sure I understand.. was this a bug and now it is fixed? or this is a proactive change to avoid a possible bug in the future.
If it is the first case, see if it is important enough to mention it in the changelog.

Reply:
Yeah. It is a NPE bug. Will add it to CHANGELOG